docs/advanced/general-privacy.md

```diff
@@ -0,0 +1,61 @@
+
+## Transaction Broadcasting
+
+Broadcasting a transaction can reveal that the transaction you are broadcasting belongs to you. A solution for this is to broadcast your transaction through a website, over Tor. For example, <https://explorer.dcrdata.org/decodetx>
+It's also important to note, that different wallets might use different [address types](address-details.md). This can also give a hint about your identity, which combined with other signals can ultimately reveal your identity.
+
+## Trade-offs of Reusing vs Not Reusing Addresses
+
+For privacy the best practice is to not reuse an address ever. A cryptocurrency address can be thought of as a cryptocurrency invoice instead. The term "address" does not mean, that it was designed to reuse it, like one would do with a bank account number for example. A privacy conscious user will use a newly generated address for every single transaction, and never reuse an address unless absolutely necessary. 
+On the other hand, there may be situations where address reuse may be preferred, or even necessary. The Decred Treasury is a good example of this. Transparency and auditability are more important than privacy in the context of the treasury.
+
+## Change Address
+
+If we only spend a fraction of an Unspent Transaction Output ([UTXO](https://coincentral.com/utxo-beginners-explainer/)), a change output will be generated. This is not good for privacy. For example, the change UTXO will be merged with another unspent transaction output next time when we are spending money, thus creating a signal that both of the transactions belong to us. An average wallet software won’t give us information about this, we won’t know, for example, that 2 UTXOs were merged to create the transaction. If you care about privacy, you should exactly know which UTXOs are being spent. [This](https://medium.com/@nopara73/coin-control-is-must-learn-if-you-care-about-your-privacy-in-bitcoin-33b9a5f224a2) is a good article on coin control.
+
+## Avoid Identification when Acquiring Cryptocurrencies 
+
+Try to avoid getting cryptocurrencies from websites that require KYC or AML, or any other means of identification. Be aware that centralized exchanges can be hacked, thus providing the collected information to the hacker, who can belong to any kind of organization. It’s also possible that at some point an oppressive government will request the collected information. ATMs can be equipped with cameras, or there can be cameras nearby the ATM. This combined with the blockchain can reveal our identity. 
+
+## Avoid Leaving Identifiers in Public Places
+
+Many times this is something physical, for example leaving behind a printed QR code somewhere, writing our nickname or real name on a piece of paper. Your name can be inprinted on the next page of the sticky notes stack, for example. Or that could also happen, that you leave behind your phone or computer unlocked for some minutes, with important information displayed on screen.
```

Comment:
I noticed the word "our" and "your" is used interchangeably throughout this file. For example it says "our nickname" here and that just seems off to me. Suggest just sticking to "your" in all cases.